firefox_puppeteer/tests/test_toolbars.py

```diff
@@ -135,3 +137,71 @@ def test_matching_text(self):
             self.assertTrue(len(all_matches) > 0)
             for match_fragment in all_matches:
                 self.assertIn(match_fragment, (input_text, input_text.upper()))
+
+
+class TestIdentityPopup(FirefoxTestCase):
+
+    def test_box(self):
+        self.assertIsNotNone(
+            self.browser.navbar.locationbar.identity_popup.box)
+
+    def test_country_label(self):
+        self.assertIsNotNone(
+            self.browser.navbar.locationbar.identity_popup.country_label)
+
+    def test_encryption_label(self):
+        self.assertIsNotNone(
+            self.browser.navbar.locationbar.identity_popup.encryption_label)
+
+    def test_encryption_icon(self):
+        self.assertIsNotNone(
+            self.browser.navbar.locationbar.identity_popup.encryption_icon)
+
+    def test_host(self):
+        self.assertIsNotNone(
+            self.browser.navbar.locationbar.identity_popup.host)
+
+    def test_is_open(self):
+        self.assertEquals(
+            self.browser.navbar.locationbar.identity_popup.is_open, False)
+
+        url = 'https://ssl-ev.mozqa.com'
```

Comment:
Reason for that is that our unit tests should touch as less as possible the external network.